src/ucp/rma/get_offload.c

```diff
@@ -93,12 +93,12 @@ ucp_proto_get_offload_bcopy_init(const ucp_proto_init_params_t *init_params)
         .super.flags         = UCP_PROTO_COMMON_INIT_FLAG_RECV_ZCOPY |
                                UCP_PROTO_COMMON_INIT_FLAG_REMOTE_ACCESS |
                                UCP_PROTO_COMMON_INIT_FLAG_RESPONSE,
-        .max_lanes           = 1,
+        .max_lanes           = context->config.ext.max_rma_lanes,
         .initial_reg_md_map  = 0,
         .first.tl_cap_flags  = UCT_IFACE_FLAG_GET_BCOPY,
         .first.lane_type     = UCP_LANE_TYPE_RMA,
         .middle.tl_cap_flags = UCT_IFACE_FLAG_GET_BCOPY,
-        .middle.lane_type    = UCP_LANE_TYPE_RMA,
+        .middle.lane_type    = UCP_LANE_TYPE_RMA_BW,
```

Comment:
do we need to align `RMA` and `RMA_BW` lanes now? Also, imo, we should count `RMA` lane as one of tx lanes, so that we will need to initialize `max_rma_lanes -1` of `RMA BW` lanes

Reply:
.first.lane_type = UCP_LANE_TYPE_RMA_BW